Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs

```diff
@@ -188,6 +188,9 @@ protected override void Draw(DrawingHandleScreen handle)
         // Frontier - collect blip location data outside foreach - more changes ahead
         var blipDataList = new List<BlipData>();
 
+        // Frontier Corvax: north line drawing
+        DrawNorthLine(handle, rot);
```

Comment:
You moved `DrawNorthLine()` after "our grid draw" which means north line will draw on layer above console's grid but below other grids.

Reply:
I believe it makes a bit of sense: I think that north direction shouldn't be covered by console's grid so it'll stay always visible but at the same time the north direction won't obstruct visibility of grids on the map itself.

But given how thin the lines on radar are, it doesn't matter too much in the end.